### PR TITLE
add ts and less translations

### PR DESCRIPTION
**Pull request by zirka28**

### CODE REVIEW

**Comment by byshovets** [comment]
Please fix readme file

**Comment by byshovets** [comment]
Make single commit, please

File: moninag/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+    <html>
+
+        <head>  
+            <tite> Moninag </title>
```
Comment (ghost) [inline moninag/templates/index.html]: Tag spelling!

File: README.md [hunk 2]
```diff
@@ -35,3 +35,14 @@ We are assuming that you're using bash & you have to install or clone such packa
         }
     }
     ```
+* Install npm and packages
+ 
+  ```
+  sudo apt-get install node-js
```
Comment (hutsulyk) [inline README.md]: Node.js install not working:
```
sudo apt-get install node-js
...
E: Unable to locate package node-js
```

Working solution

```
curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
sudo apt-get install nodejs
```

File: README.md [hunk 3]
```diff
@@ -35,3 +35,14 @@ We are assuming that you're using bash & you have to install or clone such packa
...
+  npm install -g gulp
+  npm install -g typescript
+  npm i
+  gulp
```
Comment (mhumetskyi) [inline README.md]: Remove extra line break
```
npm i gulp
```
And perhaps will be better to use same style as above (i == install) :relieved: 

```
npm install gulp
```

File: moninag/templates/index.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+	        <link rel="stylesheet" type="text/css" href="static/css/styles.css">
+		<script type="text/javascript" async="" src="static/js/source_app.js"></script>
```
Comment (ghost) [inline moninag/templates/index.html]: incorrect path for js and css files

Comment (hutsulyk) [reply moninag/templates/index.html]: Add this to settings.py
```
STATICFILES_DIRS = [
    os.path.join(BASE_DIR, "static"),
]
```

File: moninag/moninag/settings.py [hunk 5]
```diff
@@ -114,8 +115,16 @@
...
 
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, "static"),
+]
+
 STATIC_URL = '/static/'
 
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, "static"),
+]
```
Comment (mhumetskyi) [inline moninag/moninag/settings.py]: `STATICFILES_DIRS` duplicate

File: moninag/static/src/ts/app/app.module.ts [hunk 6]
```diff
@@ -0,0 +1,52 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { RouterModule }   from '@angular/router';
+
+import { AppComponent }        from './app.component'
+import { ServersComponent } from './servers.component';
+import { ServicesComponent }     from './services.component';
+import { ChecksComponent }         from './checks.component';
```
Comment (hutsulyk) [inline moninag/static/src/ts/app/app.module.ts]: Bad formatted imports

File: moninag/static/src/ts/app/app.module.ts [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  {
+    path: 'services',
+    component: ServicesComponent
+  },
+
+  {
+    path: 'checks',
+    component: ChecksComponent
+  }
+])
```
Comment (hutsulyk) [inline moninag/static/src/ts/app/app.module.ts]: Add indents
```
RouterModule.forRoot([
            {
                path: '',
                redirectTo: '/servers',
                pathMatch: 'full'
            },
            {
                path: 'servers',
                component: ServersComponent
            },

            {
                path: 'services',
                component: ServicesComponent
            },

            {
                path: 'checks',
                component: ChecksComponent
            }
        ])
```

File: moninag/home/models.py [hunk 8]
```diff
@@ -0,0 +1,3 @@
+from django.db import models
+
+# Create your models here.
```
Comment (hutsulyk) [inline moninag/home/models.py]: Unnecessary comment

File: moninag/home/tests.py [hunk 9]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (hutsulyk) [inline moninag/home/tests.py]: Unnecessary comment

File: moninag/home/admin.py [hunk 10]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
Comment (hutsulyk) [inline moninag/home/admin.py]: Unnecessary comment

File: moninag/static/src/ts/app/checks.component.ts [hunk 11]
```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'checks-app',
+  template: `<h1>Hello checks</h1>`
+})
+export class ChecksComponent {}
```
Comment (lhalam) [inline moninag/static/src/ts/app/checks.component.ts]: add end line

File: moninag/static/src/ts/app/servers.component.ts [hunk 12]
```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'servers-app',
+  template: `<h1>Hello servers</h1>`
+})
+export class ServersComponent {}
```
Comment (lhalam) [inline moninag/static/src/ts/app/servers.component.ts]: add end line

File: moninag/static/src/ts/app/services.component.ts [hunk 13]
```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'services-app',
+  template: `<h1>Hello services</h1>`
+})
+export class ServicesComponent {}
```
Comment (lhalam) [inline moninag/static/src/ts/app/services.component.ts]: add end line

File: moninag/static/src/ts/app/app.component.ts [hunk 14]
```diff
@@ -0,0 +1,16 @@
...
+    <a routerLink="/servers">Servers</a>
+    <a routerLink="/services">Services</a>
+    <a routerLink="/checks">Checks</a>
+
+    <router-outlet></router-outlet>
+  `
+})
+export class AppComponent {
+  title = 'Moninag';
+}
```
Comment (lhalam) [inline moninag/static/src/ts/app/app.component.ts]: add end line

File: moninag/home/views.py [hunk 15]
```diff
@@ -0,0 +1,8 @@
+from django.shortcuts import render_to_response
+from django.views.generic.base import View
+
+
+class IndexView(View):
+
```
Comment (lhalam) [inline moninag/home/views.py]: add section on docstrings

**Review by hutsulyk — APPROVED** [review]
Looks good!)

**Review by ghost — APPROVED** [review]
seems legit

**Pull request merged**